sdks/python/apache_beam/io/filesystem.py

```diff
@@ -330,6 +334,10 @@ def _get_compression_type(path, compression_type):
                       'was %s' % type(compression_type))
     return compression_type
 
+  @staticmethod
```

Comment:
This was in the IOChannelFactory and is used only via that in some old code. I think FS is a reasonable place for this as it is a construct introduced by the FS class. Do you have other places in mind ?